### PR TITLE
Fixed persister issue

### PR DESCRIPTION
**Pull request by lrochette**
Add catalogItem support
Remove some debug info print

### CODE REVIEW

File: META-INF/plugin.xml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
   <commander-version min="6.0" />
-  <description>A plugin to deploy your DSL code structure to your servers</description>
-  <pluginWizardBased>true</pluginWizardBased>
   <depends min="1.4.0">EC-PluginManager</depends>
+  <description>A plugin to deploy your DSL code structure to your servers</description>
   <help>help.xml</help>
+  <key>EC-DslDeploy</key>
+  <label>EC-DslDeploy</label>
+  <pluginWizardBased>true</pluginWizardBased>
+  <version>0.9.2</version>
```
Comment (urvashisingh) [inline META-INF/plugin.xml]: How about we make this version 1.0.0?

File: ecpluginbuilder.pl [hunk 2]
```diff
@@ -14,7 +15,7 @@
 
 my $epb="../ecpluginbuilder";
 
-my $pluginVersion = "0.9.1";
+my $pluginVersion = "0.9.2";
```
Comment (urvashisingh) [inline ecpluginbuilder.pl]: I guess this will also need to be bumped to v 1.0.0.

File: specs/spec-report/aggregated_report.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{"com.electriccloud.plugin.spec.NMB27865":{"executedFeatures":["NMB-27865","plugin promotion"],"ignoredFeatures":[],"stats":{"failures":0,"errors":0,"skipped":0,"totalRuns":2,"successRate":1.0,"time":24500},"title":"","narrative":""},"com.electriccloud.plugin.spec.PluginTestHelper":{"executedFeatures":[],"ignoredFeatures":[],"stats":{"failures":0,"errors":0,"skipped":0,"totalRuns":0,"successRate":1.0,"time":1761},"title":"","narrative":""},"com.electriccloud.plugin.spec.NMB27865":{"executedFeatures":["NMB-27865","plugin promotion"],"ignoredFeatures":[],"stats":{"failures":1,"errors":0,"skipped":0,"totalRuns":2,"successRate":0.5,"time":8379},"title":"","narrative":""},"com.electriccloud.plugin.spec.PluginTestHelper":{"executedFeatures":[],"ignoredFeatures":[],"stats":{"failures":0,"errors":0,"skipped":0,"totalRuns":0,"successRate":1.0,"time":1981},"title":"","narrative":""}}
```
Comment (urvashisingh) [inline specs/spec-report/aggregated_report.json]: files under specs/spec-report are test output. We don't need to check them in.

File: META-INF/plugin.xml (hunk 1 above)
Comment (lrochette) [reply META-INF/plugin.xml]: good idea

File: ecpluginbuilder.pl (hunk 2 above)
Comment (lrochette) [reply ecpluginbuilder.pl]: ecpluginbuilder.pl automatically update plugin.xml
it's not necessary for my own build but that way I'm sure the official build won't be out of version

File: specs/spec-report/aggregated_report.json (hunk 3 above)
Comment (lrochette) [reply specs/spec-report/aggregated_report.json]: Added to .gitignore
what about the specs/.gradle?

File: dsl/com/electriccloud/commander/dsl/util/BaseProject.groovy [hunk 4]
```diff
@@ -132,9 +132,10 @@ abstract class BaseProject extends DslDelegatingScript {
       pipesDir.eachDir {dlist << it}
       dlist.sort({it.name})
       for (fdir in dlist) {
+        def pipeName=fdir.name
         File dslFile = getObjectDSLFile(fdir, "pipeline")
         if (dslFile?.exists()) {
-          println "Processing pipeline DSL file ${dslFile.absolutePath}"
+          println "Processing pipeline file projects/$projectName/pipelines/$pipename/${dslFile.name}"
```
Comment (urvashisingh) [inline dsl/com/electriccloud/commander/dsl/util/BaseProject.groovy]: typo $pipename should be $pipeName

File: specs/spec-report/aggregated_report.json (hunk 3 above)
Comment (urvashisingh) [reply specs/spec-report/aggregated_report.json]: yes, that could be ignored as well

File: dsl/com/electriccloud/commander/dsl/util/BaseProject.groovy (hunk 4 above)
Comment (lrochette) [reply dsl/com/electriccloud/commander/dsl/util/BaseProject.groovy]: Fixed. Need more tests.
I ran an import successfully
Will add specs tests

**Pull request merged**